### ISSUE TITLE
[Feature Request] One-time Session ID to avoid exposing the real Session ID

### ISSUE THREAD

**Issue by x80486** [oxen-io/session-android#1231]
## Code of Conduct

- [X] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-android/blob/master/CODE_OF_CONDUCT.md).

## Description

It would be useful if there was a way to send a one-time `Session ID` (or something similar) to people we would like to connect to. In that sense, the real `Session ID` is never exposed.

Not sure how that may look like after, because I've seen that the real `Session ID` is printed when I request more information about the messages, etc.

> This is something I've seen in other applications — so I didn't come up with this idea.

**Comment by KeeJef** [comment]
Closing in favor of #109 Although it's not the exact same request, #109 is likely to be supported much sooner and if the desire for per contact Session ID's is still present after #109 we can reopen
